.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Please provide a detailed description of what was done in this PR.  
+(And mentioned if linked to an issue [docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
+
+## Changes include
+
+- [ ] Bugfix (non-breaking change that solves an issue)
+- [ ] New feature (non-breaking change that adds functionality)
+- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
```

Comment:
Please add these items to the list:
* Tests
* Documentation
* Other (for changes that might not fit in any category)

Reply:
94001c974a80b0f8f33fdef779d7a0778ebb26f5